circle.yml

```diff
@@ -11,4 +11,8 @@ test:
     - bash .ci/tests.sh:
         parallel: true
     - python3 setup.py install:
-        parallel: true
\ No newline at end of file
+        parallel: true
+
+test:
+  post:
+    - bash <(curl -s https://codecov.io/bash)
```

Comment:
That's what Lasse suggested: https://github.com/coala-analyzer/coala-quickstart/pull/24#issuecomment-234761381

But it doesn't seem to work though, does it?